rosdep/base.yaml

```
@@ -3889,6 +3889,13 @@ libgtkmm:
   fedora: [gtkmm24]
   gentoo: [dev-cpp/gtkmm]
   ubuntu: [libgtkmm-2.4-dev]
+libgtkmm3:
+  alpine: [gtkmm3]
+  arch: [gtkmm3]
+  debian: [libgtkmm-3.0-dev]
+  fedora: [gtkmm30]
+  gentoo: [dev-cpp/gtkmm:3.0]
+  ubuntu: [libgtkmm-3.0-dev]
```

So this key seems to be mixing the devel and runtime libraries in some of the distributions.  Also, we generally prefer to have the key name match the Ubuntu package name.  So I think this should end up being:

```suggestion
libgtkmm3.0-dev:
  alpine: [gtkmm3-dev]
  arch: [gtkmm3]
  debian: [libgtkmm-3.0-dev]
  fedora: [gtkmm30-devel]
  gentoo: [dev-cpp/gtkmm:3.0]
  rhel: [gtkmm30-devel]
  ubuntu: [libgtkmm-3.0-dev]
```

---

I agree to your suggestions.